README.md

```diff
@@ -1,8 +1,134 @@
 flaky-test-handler-plugin
 =========================
-This plugin is used to provide various support for handling flaky tests. It currently supports for Git and Maven.
-It includes support for the latest version of the Maven surefire plug-in which produces additional data about test 
-flakiness using the new "rerunFailingTestsCount" option. It also supports re-running  only failed tests for a 
-failed build at the exact failed Git revision. 
 
-Finally it aggregates statistics of tests (passes, fails and flakes) over Git revisions.
+[![Jenkins Plugin](https://img.shields.io/jenkins/plugin/v/flaky-test-handler.svg)](https://plugins.jenkins.io/flaky-test-handler)
+[![Jenkins Plugin Installs](https://img.shields.io/jenkins/plugin/i/flaky-test-handler.svg?color=blue)](https://plugins.jenkins.io/flaky-test-handler)
+
+## Introduction
+
+A flaky test is a test which could fail or pass for the same
+configuration. Flaky tests could be harmful for developers because their
+failures do not always indicate bugs in the code.
+
+This plugin is designed to handle flaky tests, including re-running
+failed tests, aggregate and report flaky tests statistics and so on.
+
+**Note: Currently most features of this plugin are designed for Git,
+Maven and freestyle build projects.**
```

Comment:
should we remove the _freestyle build_ part, now that you added support for pipeline? Or are there still limitations?